packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -74,7 +74,9 @@ export class DocumentsService {
       leafClusterId: params.leafClusterId,
       kubeId: params.kubeId,
       kubeUri: options.kubeUri,
-      kubeConfigName: options.kubeConfigName || `${params.kubeId}-${unique(5)}`,
+      kubeConfigName:
```

Comment:
This could become `kubeConfigPath` perhaps, what do you think?

Reply:
What would you say for `kubeConfigRelativePath`? It would match `relativePath` in `removeKubeConfig()`. `path` alone sound like a full path to the file 🤔

Reply:
`kubeConfigRelativePath` sounds fine. 👍